docs/installation/others/enabling-ssl-connection-in-ds.adoc

```diff
@@ -1,103 +1,72 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="enabling-ssl-connection-in-ds_{context}"]
-// This content was copied and adjusted from https://github.com/dogtagpki/pki/wiki/Enabling-SSL-Connection-in-DS
+
 = Enabling SSL Connection in DS 
 
-Follow this process to enable SSL connection in DS
-using a self-signed signing certificate and server certificate
-created using link:PKI-NSS-CLI.adoc[PKI NSS CLI] commands.
+Follow this process using `pki` CLI (run `man pki-client`) commands to enable SSL connection in DS.
```

Comment:
We're using `pki nss` commands instead of `pki client` commands, but we don't have man page for `pki -nss`, so we probably should just say `man pki`.